### ISSUE TITLE
Cannot load existing Account/Wallet from Trezor

### ISSUE THREAD

**Issue by felipecsl** [bitcoin-solutions/multibit-hd#859]
I have a Trezor that I was using with Multibit HD in a different laptop. I recently got a new laptop and wanted to start using Trezor on this new laptop. However, it doesn't seem to be possible to import an existing account present in my Trezor. When I go to mytrezor.com I can see my balance and previous transactions, however, Multibit HD shows an empty wallet, it seems to have created a new empty one instead of using the existing.
Is this a known bug/limitation?

Thanks!

**Comment by jim618** [comment]
MultiBit HD only supports one account in its Trezor wallet.
(This is a technical limitation, described more here: https://multibit.org/en/help/hd0.2/how-spv-works.html)

Thus is you've created multiple accounts in your Trezor wallet you'll only see the transactions in the first one.

If, on the other hand, you aren't referring to HD accounts it could be that you are looking at a regular MultiBit HD wallet.

The easiest way to check is:
1) Plug in your Trezor to your USB port first
2) Make sure myTrezor.com is NOT on your browser
3) Start up MultiBit HD

You should then see a wizard giving you the option to create a new Trezor wallet. Just run through that.

**Comment by felipecsl** [comment]
I've done the steps you enumerated multiple times and it doens't work. It must be that it's picking the wrong account then.

**Comment by jim618** [comment]
Yeah it does sound like your transactions are on the 'non-first' account, in which case MultiBit HD won't see them.
